Update setup.sh.md (#2926)
docker-mailserver/docker-mailserver: docs/content/config/setup.sh.md

        ./setup.sh debug show-mail-logs

EXAMPLES
    ./setup.sh email add test@example.com [password]
        Add the email account test@example.com. You will be prompted

>         to input a password afterwards if no password was supplied.
>         You can supply password in plain-text.

Reviewer: ```suggestion
        to input a password afterwards if no password was supplied.
        When supplying `[password]`, it should be in plaintext.
```
